src/mobly_runner/mobly_runner.py

```diff
@@ -325,7 +325,7 @@ def main() -> None:
         _padded_print('Uploading test results to Resultstore/BTX.')
         upload_args = [
             str(latest_logs), '--start_time', str(start_time), '--duration',
-            str(duration)
+            str(duration), '--abort_if_no_creds',
```

Comment:
So the mobly runner auto uploading feature requires that user should set up the local credentials first. Should we mention this in [this section of README](https://github.com/android/mobly-android-partner-tools?tab=readme-ov-file#automatically-upload-results-upon-test-completion)